src/Receiver/ReceiverChartView.tsx

```diff
@@ -33,10 +33,10 @@ const ChartView = () => {
     const lastReceived = useSelector(getLastReceived);
     const isRunning = useSelector(getIsRunning);
     const isInTransmitterMode = useSelector(getIsInTransmitterMode);
-    const [maxY, setMaxY] = useState(0);
+    const [maxY, setMaxY] = useState(10);
 
     useEffect(() => {
-        if (isRunning) setMaxY(10);
+        if (isRunning) setMaxY(0);
```

Comment:
Why were these 2 values swapped?

Reply:
Somehow the chart JS acts differently and actually both should be 0 as we relay on line 93 in the same file to set it to what ever the max y axis value is. Note this bar is only sed to show a background to the actual data bar. so I needs to have the same value as max (y-axis) of the received packets